index.html

```diff
@@ -7,48 +7,203 @@
     <link rel="stylesheet" type="text/css" href="style.css">
     <title>Ajrass Portfolio</title>
   </head>
-
   <body>
     <header>
-      <a id="logo" href="#">Ajrass</a>
-      <nav>
-        <img src="./svg/menu-button.svg" alt="menu-button">
+      <nav id="nav-bar">
+        <a class="logo" href="#">Ajrass</a>
+        <a class="menu-icon" href="#"><img src="default/SVG/IconMenu.svg" alt="Menu Icon"></a>
       </nav>
     </header>
     <main>
-      <section id="headline">
-        <h1 id="salut">Hey There.<br>I'm Ajrass.</h1>
-        <h2 id="job">I'm a Software Developer</h2>
-        <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-    don't hesitate to contact me.</p><br>
-        <ul id="links">
+      <section class="headline background">
+        <h1 class="hello">Hey There.<br>I'm Ajrass.</h1>
+        <h2 class="iam">I'm a Software Developer</h2>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hesitate to contact me.
+        </p>
+        <ul class="social">
           <li>
             <a class="link" href="https://github.com/tajemouti" rel="noopener noreferrer" target="_blank">
-            <img src="./svg/github.svg" alt="GitHub">
+              <img src="default/SVG/IconGitHub.svg" alt="GitHub">
             </a>
           </li>
           <li>
             <a class="link" href="https://www.linkedin.com/in/ajrass" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/linkedin.svg" alt="LinkedIn">
+              <img src="default/SVG/Icon Linkedin.svg" alt="LinkedIn">
             </a>
           </li>
           <li>
             <a class="link" href="https://wellfound.com/u/ajrass-tajemouti" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/wellfound.svg" alt="WellFound">
+              <img src="default/SVG/IconAngelist.svg" alt="WellFound">
             </a>
           </li>
           <li>
             <a class="link" href="https://twitter.com/AjrassTajemouti" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/twitter.svg" alt="Twitter">
+              <img src="default/SVG/IconTwitter.svg" alt="Twitter">
             </a>
           </li>
           <li>
             <a class="link" href="https://medium.com/@tajemouti/" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/medium.svg" alt="Medium">
+              <img src="default/SVG/IconMedium.svg" alt="Medium">
             </a>
           </li>
         </ul>
       </section>
+      <section class="work-section">
+        <span>
+          <h2 class="work-section-title">My Recent Works</h2>
+          <hr class="divider">
```

Comment:
- [ ] Please 🙏🏽 use the correct tag for the best practices of HTML & CSS, use `div` instead of `hx`

Reply:
Thank you for your advice.. I appreciated..